.travis.yml

```diff
@@ -1,13 +1,18 @@
-language: python
+language: minimal
 
 sudo: false
 
 matrix:
+  fast_finish: true
   include:
-    - env: ENV_FILE="ci/travis/36-latest-conda-forge.yaml"
-    - env: ENV_FILE="ci/travis/37-latest-conda-forge.yaml"
+    - name: "python-3.6"
+      env: PY=3.6
+    - name: "python-3.7"
+      env: PY=3.7
+    - name: "tarball"
+      env: PY=3.7
```

Comment:
This test matrix implements the two tests that existed, for 3.6 and 3.7, and add a new one to check the `tarball` created. This will help prevent some release issues.